### ISSUE TITLE
Compile error on ubuntu with clang!

### ISSUE THREAD

**Issue by jinwenPonyAI** [ocornut/imgui#2324]
**ImGui Version:** 1.67

**Description:**
When compile ImGui-1.67 the following compile error is emitted:
`Unknown warning group - '-Wzero-as-null-pointer-constant', ignored`

**Suggested fix:**
``````
#if !defined(__has_warning) || __has_warning("-Wzero-as-null-pointer-constant")
#pragma clang diagnostic ignored "-Wzero-as-null-pointer-constant"
#endif
``````

**Comment by ocornut** [comment]
This should already be fixed, see #2299

**Comment by jinwenPonyAI** [comment]
Thanks a lot, ocornut.

**Comment by jinwenPonyAI** [comment]
Could you merge this fix into imgui-1.67? I am using a build system that grab code src from git periodically. For the stability reason,  I can't use your master...
